auth/routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint, request, jsonify
+from auth.utils import authenticate, generate_token
+
+auth_bp = Blueprint('auth', __name__)
+
+@auth_bp.route('/login', methods=['POST'])
+def login():
+    data = request.get_json()
+    user = authenticate(data.get('email'), data.get('password'))
+    if user:
```

Comment:
data might be null here. check please
```suggestion
    data = request.get_json()
    if data:
        user = authenticate(data.get('email'), data.get('password'))
        if user:
```